<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createstructurewindow</class>
 <widget class="QWidget" name="createstructurewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Structure</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Structure Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Symmetry</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>671</width>
       <height>251</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Atoms</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Symbol</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>V1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>V2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>V3</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>672</width>
       <height>251</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Lattice Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>45</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>V1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>V2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>V3</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>X</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Y</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Z</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hexagonal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rectangular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cubic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_4">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_5">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>createstructurewindow</receiver>
   <slot>cancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>934</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>createstructurewindow</receiver>
   <slot>plusButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>createstructurewindow</receiver>
   <slot>minButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>createstructurewindow</receiver>
   <slot>editCoord(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>createstructurewindow</receiver>
   <slot>editName(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>createstructurewindow</receiver>
   <slot>createButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>createstructurewindow</receiver>
   <slot>comboBoxSelection(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget_2</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>createstructurewindow</receiver>
   <slot>updateCell(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addAtom(Particle)</slot>
  <slot>editAtom(QTableWidgetItem*)</slot>
  <slot>editName(QString)</slot>
  <slot>editCoord(QTableWidgetItem*)</slot>
  <slot>updateCell(int,int)</slot>
  <slot>plusButtonClicked()</slot>
  <slot>minButtonClicked()</slot>
  <slot>createButtonClicked()</slot>
  <slot>cancelButtonClicked()</slot>
  <slot>comboBoxSelection(QString)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
